src/viam/logging.py

```diff
@@ -32,6 +32,7 @@ def __init__(self, parent: "RobotClient"):
         try:
             self.loop = asyncio.get_event_loop()
         except RuntimeError:
+            self._logger.warn("Created an event loop from a new thread. We recommend initializing loggers in the main module thread.")
```

Comment:
[super nit] "Created" -> "Creating" since we haven't created the event loop yet since this log comes before that LOL just in case an error happens